src/main/java/com/sample/Calculator.java

```diff
@@ -0,0 +1,20 @@
+package com.sample;
+
+public class Calculator
+{
+    public static void main( String[] args ){
+        System.out.println(sum(9,6));
+    }
+
+    public static int sum(long a, long b){
```

Comment:
Use the same type for arguments and  result or at least don't make result less size than arguments. 
something like that : https://hg.openjdk.java.net/jdk8/jdk8/jdk/file/687fd7c7986d/src/share/classes/java/lang/Math.java#l864

Or any other validation you can came up with, even if it significantly restricts valid input. And add restriction for arguments into java doc. So that who eve want to use your class can understand limitations.